src/main/java/htsjdk/samtools/util/BlockCompressedInputStream.java

```diff
@@ -57,10 +57,12 @@
     public final static String INCORRECT_HEADER_SIZE_MSG = "Incorrect header size for file: ";
     public final static String UNEXPECTED_BLOCK_LENGTH_MSG = "Unexpected compressed block length: ";
     public final static String PREMATURE_END_MSG = "Premature end of file: ";
-    public final static String CANNOT_SEEK_STREAM_MSG = "Cannot seek on stream based file ";
```

Comment:
I did, it's on the line below. This is the new one. I can reverse the order to make the diff easier to view.